tests/package.json

```diff
@@ -14,6 +14,6 @@
 		"@rbxts/roact": "github:roblox-ts/roact",
 		"@rbxts/services": "^1.5.3",
 		"@rbxts/testez": "^0.4.2-ts.0",
-		"@rbxts/types": "^1.0.751"
+		"@rbxts/types": "^1.0.794"
```

Comment:
Please revert `tests/package.json` and `tests/package-lock.json` changes. Not relevant to this PR.